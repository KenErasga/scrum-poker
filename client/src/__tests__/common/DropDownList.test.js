import React from 'react';
import renderer from 'react-test-renderer';
import DropDownList from '../../common/DropDownList';

it('DropDownList renders correctly', () => {
  const tree = renderer
    .create(<DropDownList estimate='1' numberList={[1,2,3]} setEstimate={(e) => e}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});