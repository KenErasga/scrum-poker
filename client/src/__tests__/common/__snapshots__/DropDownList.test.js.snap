// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDownList renders correctly 1`] = `
<div>
  <div
    className="MuiFormControl-root makeStyles-formControl-1"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
      data-shrink={true}
      id="estimateLabel"
    >
      Estimate
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <select
        aria-invalid={false}
        className="MuiNativeSelect-root MuiNativeSelect-select MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        value="1"
      >
        <option
          value={1}
        >
          1
        </option>
        <option
          value={2}
        >
          2
        </option>
        <option
          value={3}
        >
          3
        </option>
      </select>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiNativeSelect-icon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </div>
</div>
`;
